From ae29bcf2ddf2b82098d003768a51d28b90ffc5b4 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Tue, 25 Feb 2020 16:53:31 +0000
Subject: [PATCH] ARM: dts: Remove virtgpio from bcm2711-rpi-4-b.dts

Pi 4B doesn't need the virtgpio driver (it was added to reduce the
overhead of driving the ACT LED on the Pi 3B's GPIO expander), so
remove the instantiation by the Pi 4 DTS file to avoid confusion.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/bcm2711-rpi-4-b.dts | 10 ----------
 1 file changed, 10 deletions(-)

--- a/arch/arm/boot/dts/bcm2711-rpi-4-b.dts
+++ b/arch/arm/boot/dts/bcm2711-rpi-4-b.dts
@@ -34,16 +34,6 @@
 	};
 };
 
-&soc {
-	virtgpio: virtgpio {
-		compatible = "brcm,bcm2835-virtgpio";
-		gpio-controller;
-		#gpio-cells = <2>;
-		firmware = <&firmware>;
-		status = "okay";
-	};
-};
-
 &mmcnr {
 	pinctrl-names = "default";
 	pinctrl-0 = <&sdio_pins>;
